nntrainer/models/neuralnet.cpp

```diff
@@ -391,6 +394,7 @@ sharedConstTensors NeuralNetwork::forwarding(
       model_graph.LoadTensors(f);
       model_graph.checkLoadComplete(f);
       node->forwarding(training);
+      model_graph.Inactive(f);
```

Comment:
leave the comment why we need this Inactive. or name the function to know what this function for.